src/main/java/com/rackspace/ceres/app/services/DataTablesStatements.java

```diff
@@ -88,10 +91,9 @@ private void buildRawStatements(AppProperties appProperties) {
         + " WHERE " + TENANT + " = ?"
         + "  AND " + TIME_PARTITION_SLOT + " = ?"
         + " AND series_set_hash = ?";
-    rawGetHashQuery =
-        "SELECT series_set_hash FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
-            + " WHERE " + TENANT + " = ?"
-            + "  AND " + TIME_PARTITION_SLOT + " = ?";
+    rawGetHashQuery = "SELECT series_set_hash FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
```

Comment:
You can declere these as separate private final static variables as strings with placeholders. Also I heard this is better for performance as well.

See example:

String s = "hello %s!";
s = String.format(s, "world");
assertEquals(s, "hello world!"); // should be true

Reply:
made changes in PR https://github.com/racker/ceres/pull/43